docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -75,6 +75,50 @@ ALTER TABLE table_identifier ADD COLUMNS ( col_spec [ , ... ] )
 
     Specifies the columns to be added.
 
+### DROP COLUMNS
+
+`ALTER TABLE DROP COLUMNS` statement drops mentioned columns from an existing table.
+Note that this statement is only supported with v2 tables.
+
+
+#### Syntax
+
+```sql
+ALTER TABLE table_identifier DROP COLUMNS ( col_name1, ... )
+```
```

Comment:
Here is the syntax:
```
    | ALTER TABLE multipartIdentifier
        DROP (COLUMN | COLUMNS)
        '(' columns=multipartIdentifierList ')'                        #dropTableColumns
    | ALTER TABLE multipartIdentifier
        DROP (COLUMN | COLUMNS) columns=multipartIdentifierList        #dropTableColumns
```
So I think it's something like this:
```
ALTER TABLE table_identifier DROP { COLUMN | COLUMNS } [ ( ] col_name [ , ... ]  [ ) ]
```
I guess add a parameter in the below section for `col_name`.

You might want to change the add column syntax too to make it more accurate.

Reply:
I just realized that previously we used `col_spec` for add column. I think `col_name` is more accurate, because `col_spec` makes me think it is column name + data type. Could you please check if other doc use `col_spec` too? If yes, we probably want to use `col_spec` to be consistent.

Reply:
It seems that `col_spec` is only used in this doc (for ADD COLUMN, RENAME COLUMN and ALTER/CHANGE COLUMN), and I agree with you that `col_spec` sounds misleading when only a name of a column is needed. I'll change all `col_spec` in `col_name` except ones in ADD COLUMN (since ADD COLUMN needs data type as well).